example/index.js

```diff
@@ -1,3 +1,4 @@
+import 'react-native-get-random-values';
```

Comment:
we need to document this in the installation instructions for new react native versions

Reply:
See https://github.com/voize-gmbh/reakt-native-toolkit/pull/53/commits/f8b2b25b9a08d58d9e3e44f08bc22d080f72f90b#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R411-R424